core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -150,7 +150,7 @@ private OutputStream decorate(OutputStream os) {
             try {
                 os = f.decorateLogger(this,os);
             } catch (IOException|InterruptedException e) {
-                LOGGER.log(Level.WARNING, "Failed to filter log with "+f, e);
+                logger.log(Level.WARNING, "Failed to filter log with "+f, e);
```

Comment:
there were two loggers in SlaveComputer
line 770:  private static final Logger logger = Logger.getLogger(SlaveComputer.class.getName());
line 861:  private static final Logger LOGGER = Logger.getLogger(SlaveComputer.class.getName());

I removed the uppercase definition and replaced reference to another one.